product_supplierinfo_qty_multiplier/demo/product_product.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
The file should be named by convention `product_product_demo.xml`. And the same of the others.

Reply:
done. Thanks.